### ISSUE TITLE
IFDB's logged-out home page should deprioritize updates, and instead prioritize good games

### ISSUE THREAD

**Issue by dfabulich** [iftechfoundation/ifdb-suggestion-tracker#18]
When you visit the IFDB home page for the first time (e.g. in an incognito/private window), the first/primary section of the page is "New on IFDB." This section doesn't even necessarily show new _games_ on IFDB, (new "listings") but also new reviews of old games.

If this is your first time at IFDB, there's a good chance that you're new to IF. We almost certainly don't want to direct IF newbies to the newest games/listings/reviews; instead we want to direct them to the best games we have to offer.

Concretely, I'm imagining that IFDB home page should be:

* IFDB Top Games: It should show the top 10 games sorted by reviews. (But not just sorted by average; see issues #16 and #17.)
* "New to IF?" and there it should show the top 10 games with the "recommended for beginners" tag; we'd ensure that each of them have enabled their "Play Online" button
* New on IFDB
* Vote!

Perhaps logged-in users could click a button to collapse the "Top Games" / "New to IF" section so they can navigate to the news/votes more quickly…?

**Comment by brirush84** [comment]
A possible issue here is that I suspect most users of IFDB, even frequent users, don't visit logged in. Hiding the 'new' feed could have an overall negative effect, since the top games almost never change and the front page would be static.

So I think it would be useful if the either the 'new games/reviews' feed shares equal prominence with the old, or if there is a page with a simple url (like ifdb.org/latest) that people can access to go directly to the new feed.

**Comment by dfabulich** [comment]
Good point! I note that IMDB (if you scroll down past their first-party content) shows a "popular this week" section, which normally has no overlap with the IMDB Top 100. We could probably do something similar.
